### DISCUSSION TITLE
Is it possible to retry a failed process in the same work directory?

### DISCUSSION THREAD

**Discussion by Midnighter** [nextflow-io/nextflow#2385] [Q&A] [answered]
Specifically, I have a tool (NCBI's prefetch) for downloading sequencing data from the SRA. If the download is interrupted and prefetch exits, it is possible to restart it in the same directory and it will continue the download instead of starting from scratch. Hence, I wonder if this is possible to achieve with nextflow? Can I somehow make nextflow retry the failed process in the existing work directory so that it can resume the download?

Thank you for any ideas.

**Comment by pditommaso** [comment, marked as answer]
This is not allowed on purpose because on failure the task work dir can be into an inconsistent state (missing or incomplete files, etc). 

You should wrap your  download command with a Bash script that detected the failure and recover the download.
